WolvenKit.App/Models/LaunchProfile.cs

```diff
@@ -16,9 +16,13 @@ public class LaunchProfile
     // install script files
 
     [Category("Install")]
-    [Display(Name = "Clean packed directory completely first")]
+    [Display(Name = "Clean packed directory before build?")]
     public bool CleanAll { get; set; } = true;
 
+    [Category("Install")]
+    [Display(Name = "Clean packed directory after build?")]
```

Comment:
what's the point of this exactly? :D why clean after a build?

Reply:
because Nola complained that NPV projects clutter up her disk, and tbh she has a point